nodes/Decodo/services/parameter-transformer.ts

```diff
@@ -23,22 +31,25 @@ export class PropertyHandler {
     const out: Record<string, unknown> = {};
 
     for (const prop of PropertyHandler.properties) {
-      const { name } = prop;
-      out[name] = getValue(name, 0, prop.default);
+      const { name, type } = prop;
+      const fallback = type === 'boolean' ? false : prop.default;
+      out[name] = getValue(name, 0, fallback);
     }
 
     return out;
   };
 
   static transformToScrapingParameters = (params: ScraperApiParams) => {
-    const { target, url, query, headless, geo, markdown } = params;
     return {
-      target,
-      ...(url ? { url } : {}),
-      ...(query ? { query } : {}),
-      ...(headless ? { headless: 'html' } : {}),
-      ...(geo ? { geo } : {}),
-      ...(markdown ? { markdown } : {}),
+      target: params.target,
+      ...(params.url ? { url: params.url } : {}),
+      ...(params.query ? { query: params.query } : {}),
+      ...(params.headless ? { headless: 'html' } : {}),
+      ...(params.parse ? { parse: params.parse } : {}),
+      ...(params.geo ? { geo: params.geo } : {}),
+      ...(params.locale ? { locale: params.locale } : {}),
+      ...(params.results_limit ? { limit: params.results_limit } : {}),
+      ...(params.markdown ? { markdown: params.markdown } : {}),
```

Comment:
```suggestion
      ...(params.markdown && { markdown: params.markdown }),
```

same for the lines above